packages/sdk/test/bench/splay_tree.bench.ts

```
@@ -0,0 +1,96 @@
+import { describe, bench } from 'vitest';
+import { SplayNode, SplayTree } from '@yorkie-js-sdk/src/util/splay_tree';
+import editTraceData from './editing-trace.json';
+
+class StringNode extends SplayNode<string> {
+  public removed: Boolean = false;
```

Biome

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)